lib/cats/basemeow.rb

```diff
@@ -0,0 +1,21 @@
+module Cats
+  class Basemeow
```

Comment:
I like the way you have pulled this functionality out into a base class ✅ 

But is this the right class name?

When I read the code, it seems like this class represents an entity that has the ability to `#say` something. In that case, what do you think of a name like `class Speaker`?
(with such a change you might also rename the accessor `meow` to `word` or `phrase`)